endpoint/model.json

```diff
@@ -279,45 +297,52 @@
                       "name": "deployed",
                       "type": "boolean",
                       "description": "If `true`, the endpoint metadata represents a public, live endpoint that is available for communication and a strict validator MAY test the liveness of the endpoint",
-                      "required": true,
+                      "required": false,
                       "default": false
                     },
-
                     "topic": {
                       "name": "topic",
```

Comment:
I wonder if we should allow "name" to be optional on requests but then require the server to fill it in from the key? Just to make life easier for people. We do something similar for IDs already.

Reply:
https://github.com/xregistry/spec/issues/304